server/src/main/java/io/spine/server/procman/LifecycleRules.java

```diff
@@ -63,8 +63,8 @@ public LifecycleRules() {
     /**
      * Configures the repository to archive its entities when certain events/rejections occur.
      *
-     * <p>Subsequent calls to this method do not clear the previously added event classes (i.e. not
-     * override each other).
+     * <p>Subsequent calls to this method do not clear the previously added event classes,
+     * for example, in order not to override each other.
```

Comment:
Isn't it “that is” in this context? “For example” doesn't feel right here. The same comment applies to the similar case below.